website/content/en/docs/reference/metrics.md

```diff
@@ -130,12 +130,12 @@ Number of nodeclaims registered in total by Karpenter. Labeled by the owning nod
 ### `karpenter_nodeclaims_terminated`
 Number of nodeclaims terminated in total by Karpenter. Labeled by reason the nodeclaim was terminated and the owning nodepool.
 
-## Nodepools Metrics
+## Nodepool Metrics
```

Comment:
nit: we reference NodePool with camel case elsewhere
```suggestion
## NodePool Metrics
```

Reply:
This is auto-gened and it's based off of the name of the namespace in code, so I'm hesitant to want to change this here.